docs/devops-guide/quickstart.md

```diff
@@ -17,6 +17,8 @@ _Note_: Many of the installation steps require `root` or `sudo` access.
 You will need the following packages:
 * `gnupg2`
 * `sudo` # only needed if you use sudo
+* `openjdk-8-jdk`
```

Comment:
There is one problem though.
This package is not available in Debian Buster (see: https://packages.debian.org/stretch/openjdk-8-jdk).

Reply:
[jitsi-videobridge2 package](https://github.com/jitsi/jitsi-videobridge/blob/master/debian/control) has the following pre-dependencies:
```
Pre-Depends: java8-runtime-headless | java8-runtime | java11-runtime-headless | java11-runtime
```

https://packages.ubuntu.com/bionic/java8-runtime-headless
https://packages.ubuntu.com/bionic/java8-runtime
https://packages.ubuntu.com/bionic/java11-runtime-headless
https://packages.ubuntu.com/bionic/java11-runtime

I tried hard but could not find a package which provides IBM java through those pre-deps.
Is there anything I am missing? Are these IBM deps coming from multiverse repo?

I'd suggest not to add an over-specific dependency which will sooner or later become outdated. openjdk8 is already outdated for current Ubuntu/Debian stable releases.

Just add a hint to make sure openjdk should be used, which is the default on most Debian/Ubuntu installations

Reply:
The point is, that some Jitsi components are (or were? I don't know the current status) not completely compatible to Java 11, so this pull request wanted to implement instructions to install Java 8 instead.

Nonetheless you are right, the Team should get all Jitsi Components to use Java 11 instead.
Sadly there is not much transparency about this.